subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -8007,10 +8007,12 @@ static inline void event_len_prep(struct connection *conn)
 		LL_ASSERT(free_count_rx <= 0xFF);
 
 		if (_radio.packet_rx_data_count != free_count_rx) {
-			/** TODO another role instance has obtained
-			 * memory from rx pool.
+			/* NOTE: Another role instance has obtained
+			 * memory from rx pool, skip this event and
+			 * check in the next event if resize can be
+			 * done.
 			 */
-			LL_ASSERT(0);
+			return -EAGAIN;
```

Comment:
This doesn't look right. You changed the return value of this function to u32_t but here you're returning a negative value?

Reply:
Oversight, will fix it. Thank you.